extensions/openwsman/processors/SourceInitiatedSubscriptionListener.cpp

```diff
@@ -563,7 +563,7 @@ bool SourceInitiatedSubscriptionListener::Handler::handleSubscriptionManager(str
     ws_xml_add_child(subscribe_node, XML_NS_EVENTING, WSEVENT_EXPIRES, millisecondsToXsdDuration(processor_.subscription_expiration_interval_).c_str());
 
     // Body/Filter
-    WsXmlNodeH filter_node = ws_xml_add_child(subscribe_node, XML_NS_WS_MAN, WSM_FILTER, processor_.xpath_xml_query_.c_str());
+    // WsXmlNodeH filter_node = ws_xml_add_child(subscribe_node, XML_NS_WS_MAN, WSM_FILTER, processor_.xpath_xml_query_.c_str());
```

Comment:
I believe this might be an accidental behavior change. `ws_xml_add_child` has side effects, it adds a child to the parameter node `subscribe_node`.

Reply:
I believe you're right. Readded in 72ecc8d8d60244e8087510f05fa7399324b215cb